src/string_utils.cpp

```diff
@@ -345,7 +345,7 @@ size_t StringUtils::get_num_chars(const std::string& s) {
     return j;
 }
 
-Option<bool> StringUtils::tokenize(std::string filter_query, std::queue<std::string>& tokens) {
+Option<bool> StringUtils::tokenize(const std::string& filter_query, std::queue<std::string>& tokens) {
```

Comment:
The name `tokenize` is pretty generic: maybe call it `tokenize_filter_query`? LGTM otherwise.